lib/Wasmtime.hs

```diff
@@ -179,6 +181,17 @@ import Foreign.Storable (Storable, peek, peekElemOff, poke, pokeElemOff)
 import System.IO.Unsafe (unsafePerformIO)
 import Type.Reflection (TypeRep, eqTypeRep, typeRep, (:~~:) (HRefl))
 
+--------------------------------------------------------------------------------
+-- Typedefs
+--------------------------------------------------------------------------------
+
+type Offset = Word64
+
+type Length = Word64
```

Comment:
Super nitpicking but wasmtime calls the length of the memory "size" so what about:
```suggestion
type Size = Word64
```
?